Documentation in help format

tessarin opened the pull request:
This PR adds documentation (extracted from the README file) in Vim's help format.

tessarin committed:
Add documentation in help format

This commit adds a translation of the README file to the help format.

tessarin committed:
Fix typo on Airline setting

tessarin committed:
Resolves #5

ghifarit53 commented:
Hi @tessarin, thanks for the pull

Would you please edit `daycula.txt` to reflect the latest README
I've just made a fix for the background transparency problem (#8)

tessarin committed:
Add transparency note and ports section

Two changes to documentation are made here:
 - Adds a note about issue #8 in the transparency option
 - Adds a section dedicated to the terminal ports

tessarin commented:
@ghifarit53, I added the note and also added a new section to list the terminal ports. Let me know if it is ok. Thanks!

ghifarit53 commented:
@tessarin Nice! Thanks for the change, gonna merge now
